zigzag/include/zigzag.h

```diff
@@ -4,6 +4,14 @@
 #include <wlr/render/wlr_texture.h>
 #include <wlr/util/box.h>
 
+enum zigzag_anchor {
```

Comment:
I think it is better to have `zigzag_horizontal_anchor` and `zigzag_vertical_anchor` separately. How do you think?

Reply:
I considered, but their name looks too long for me

Reply:
Alright, sounds reasonable, let's proceed with this design for now